iidm/iidm-mergingview/src/main/java/com/powsybl/iidm/mergingview/MergedLine.java

```diff
@@ -493,12 +497,14 @@ public void remove() {
 
     @Override
     public <E extends Extension<Line>> E getExtension(final Class<? super E> type) {
-        throw MergingView.createNotImplementedException();
+        return null;
+        // throw MergingView.createNotImplementedException();
```

Comment:
Just for me, could you explain why you commented the throwing of exception here? Was it only to test?

Reply:
I put two TODOs in these methods to explain why I prefer returning null or an empty list instead of throwing an exception.